ui/src/components/formA11y/Header.tsx

```diff
@@ -0,0 +1,61 @@
+// @ts-nocheck
+import styled from 'styled-components';
+import Steps from './Steps';
+
+export const Header = ({ currentLayer }) => {
+    return (
+        <Container>
+            <LogoContainer>
+                <img src="/logo.svg" />
+            </LogoContainer>
+            <StepsContainer>
+                <Steps current={currentLayer} stepInfo={stepInfo} />
+            </StepsContainer>
+        </Container>
+    );
+};
+
+const stepInfo = [
```

Comment:
Is `stepInfo` ever meant to change while we're running the application?

Reply:
These are the sub-steps for remediating the form. They will stay the same. We can move the array to another place as well. Feel free to update any of the components as needed.